backend/spherre/app/extensions.py

```
@@ -0,0 +1,10 @@
+from flask_cors import CORS
+from flask_jwt_extended import JWTManager
+from flask_migrate import Migrate
+from flask_sqlalchemy import SQLAlchemy
+from marshmallow import Schema
```

Ruff (0.11.9)

`marshmallow.Schema` imported but unused

Remove unused import: `marshmallow.Schema`

(F401)